src/components/loadable-item.cjsx

```diff
@@ -18,9 +18,25 @@ module.exports = React.createClass
     renderItem: React.PropTypes.func.isRequired
     saved: React.PropTypes.func
     load: React.PropTypes.func
+    renderLoading: React.PropTypes.func
+    update: React.PropTypes.func
+
+  componentDidMount: -> @reload({})
+  componentDidUpdate: (oldProps) -> @reload(oldProps)
```

Comment:
!  that's pretty cool. like [this](https://github.com/openstax/tutor-js/blob/master/src/flux/task-step.coffee#L17-L20)?   this could be part of the crud store config, and then
```
  componentWillUnmount: -> @props.store.forceReload()
```